schema/html5/common.rnc

```diff
@@ -210,6 +210,11 @@ common.attrs.other =
 			w:string "autofocus" | w:string ""
 		}
 
+	common.attrs.blocking = 
+		attribute blocking {
+			w:string "render" | w:string ""
```

Comment:
```suggestion
			w:string "render"
```
Per https://html.spec.whatwg.org/multipage/urls-and-fetching.html#blocking-attribute the empty string is not actually allowed as a `blocking` value, right?

Reply:
I think it is. [Blocking attribute definition](https://html.spec.whatwg.org/multipage/urls-and-fetching.html#blocking-attributes) says "A [blocking attribute](https://html.spec.whatwg.org/multipage/urls-and-fetching.html#blocking-attribute) must have a value that is an [unordered set of unique space-separated tokens](https://html.spec.whatwg.org/multipage/common-microsyntaxes.html#unordered-set-of-unique-space-separated-tokens)", while definition of set is "A set of space-separated tokens is a string containing zero or more words (known as tokens) separated by one or more [ASCII whitespace](https://infra.spec.whatwg.org/#ascii-whitespace), where words consist of any string of one or more characters, none of which are [ASCII whitespace](https://infra.spec.whatwg.org/#ascii-whitespace).". So zero words seems to be a valid value of blocking attribute. Do you agree?

Reply:
```suggestion
			list { w:string "render" }
```
This discussion made me realize that the checker schema shouldn’t be restricting the value to string, but instead needs to allow a list — since the spec says it can be a set of space-separated tokens, not just a single string.

And because there’s no way for a set of space-separated tokens to include the empty string, it wouldn’t make any sense anyway for the checker schema to include the empty string in the set of tokens allowed in the list.

And on top of that, the spec doesn’t just say it’s “unordered set of unique space-separated tokens” — instead it says it’s an “unordered set of unique space-separated tokens _**each of which are [possible blocking tokens](https://html.spec.whatwg.org/multipage/urls-and-fetching.html#possible-blocking-token)**_”

And the definition above that says:

> The operations that can be blocked are represented by **possible blocking tokens**, which are strings listed by the following table:

…and that table lists only one single allowed “possible blocking token”, which is `"render" `. It explicitly does not list the empty string as a “possible blocking token” — and the spec wouldn’t make any sense if it did allow the empty string as a token, because there’s no way for a set of space-separated tokens to include the empty string.

Reply:
That sounds good. I have commited your suggested change.